src/main/antlr/StitchingDSL.g4

```diff
@@ -26,9 +26,17 @@ serviceName: NAME;
 
 topLevelField: NAME;
 
+sourceObjectField : '$source' '.' name ;
```

Comment:
I am happy with any naming changes or different way of expressing this in the grammar. Let me know if you have better ideas.